From: Stefano Zacchiroli <zack@debian.org>
Date: Sat, 12 Feb 2011 10:17:31 +0100
Subject: Change standard directories to conform our own policy

Patch upstream findlib.conf.in so that:
 - default "manual" installation dir is /usr/local/lib/ocaml/<version>
 - findlib path look first in /usr/local/lib... and then in the
   standard ocaml library: this permit local overrides of libraries

Forwarded: not-needed
Signed-off-by: Stephane Glondu <steph@glondu.net>
---
 findlib.conf.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/findlib.conf.in b/findlib.conf.in
index 261d2c8..05f6c3d 100644
--- a/findlib.conf.in
+++ b/findlib.conf.in
@@ -1,2 +1,2 @@
-destdir="@SITELIB@"
-path="@SITELIB@"
+destdir="/usr/local/lib/ocaml/@OCamlABI@"
+path="/usr/local/lib/ocaml/@OCamlABI@:@SITELIB@:@SITELIB@/METAS"
-- 
